api/src/pcapi/routes/backoffice/non_payment_notices/forms.py

```diff
@@ -0,0 +1,76 @@
+from flask import flash
+from flask_wtf import FlaskForm
+from wtforms.validators import DataRequired
+
+from pcapi.core.offerers import models as offerers_models
+from pcapi.models import db
+from pcapi.routes.backoffice import filters
+from pcapi.routes.backoffice.forms import fields
+from pcapi.routes.backoffice.forms import utils as forms_utils
+
+
+class CreateNonPaymentNoticeForm(FlaskForm):
+    class Meta:
+        locales = ["fr_FR", "fr"]
+
+    date_received = fields.PCDateField("Date de réception de l'avis")
+
+    notice_type = fields.PCSelectField(
+        "Type d'avis",
+        choices=forms_utils.choices_from_enum(offerers_models.NoticeType, formatter=filters.format_notice_type),
+        validators=[DataRequired("Information obligatoire")],
+    )
+
+    amount = fields.PCDecimalField(
+        "Montant total dû à l'acteur culturel (en euros)",
+        use_locale=True,
+        validators=[DataRequired("Information obligatoire")],
+    )
+
+    reference = fields.PCStringField("Référence")
+    emitter_name = fields.PCStringField("Nom de l'émetteur")
+    emitter_email = fields.PCEmailField("Adresse mail de l'émetteur")
```

Comment:
Par cohérente avec partout dans le BO
```suggestion
    emitter_email = fields.PCEmailField("Adresse email de l'émetteur")
```